Dockerfile

```diff
@@ -60,6 +60,22 @@ RUN INSTALL_PKGS="nss_wrapper make gcc vim-common kernel-headers zlib-devel libc
     rpm -V $INSTALL_PKGS && \
     dnf clean all -y
 
+RUN apt-get update \
+    && apt-get install -y wget
+
+RUN mkdir -p walg && \
+    pushd walg && \
+    wget https://github.com/wal-g/wal-g/releases/download/v0.2.15/wal-g.linux-amd64.tar.gz && \
+    tar -zxvf wal-g.linux-amd64.tar.gz && \
+    ./wal-g --help && \
+    popd && \
+    /usr/libexec/fix-permissions /walg && \
+    /usr/libexec/fix-permissions /walg_scripts
+
+COPY walg/ .
+COPY walg_scripts/ .
```

Comment:
If you move your wal-g sripts in this repo's `root/usr/bin` folder, all you need to do here is to extract the wal-g archive and move it there.
You should do that after the `COPY root /` command below though, and all you need to do is 
```
RUN curl https://ftp.postgresql.org/pub/source/v11.4/postgresql-11.4.tar.gz | tar xz &&\
    mv wal-g /usr/bin

```